pkgs/by-name/ib/ibm-plex/package.nix

```diff
@@ -1,28 +1,59 @@
-{ lib, stdenvNoCC, fetchzip }:
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+  families ? [ ],
+}:
+let
+  version = import ./version.nix;
+  availableFamilies = import ./hashes.nix;
 
-stdenvNoCC.mkDerivation rec {
+  selectedFamilies =
+    if (families == [ ]) then
```

Comment:
Honestly I think we should make `families` a lambda that takes the list of families and returns a subset of it — would be much simpler logically since the input will just default to `x: x`